.travis.yml

```diff
@@ -4,16 +4,15 @@ go_import_path: go.uber.org/goleak
 
 go:
   # TODO(prashant): Enable once we have fixed Linux issues with old versions.
```

Comment:
I guess you don't need this comment any more

Reply:
:+1: